content/community/depression.md

```diff
@@ -42,7 +42,7 @@ Please understand that although we are here to provide peer support, our channel
 
 
 ## Channel Rules & Guidelines {#guidelines}
-(Last updated: December 19th, 2018)
+(Last updated: October 13th, 2019)
```

Comment:
Is this date field necessary? All pages have a footer with the modification info